docs/architecture/account.md

```diff
@@ -41,6 +41,23 @@ An `Account` ID is considered invalid if:
 - The anchor epoch exceeds $2^{16}-1$.
 - The least significant 8 bits of the ID are nonzero.
 
+An `Account` ID can be encoded in different formats:
+
+1. [**Bech32**](https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki) (default):
```

Comment:
I would mention at least something like "We strongly recommend encoding account IDs in bech32 for all user-facing applications" (or something to that effect). As it is now, as a developer I wouldn't be 100% sure what to use, so I would make it crystal clear.